examples/gke/cluster.yaml

```diff
@@ -98,14 +99,6 @@ spec:
             cpu: 30
             memory: 115G
         placement:
-          nodeAffinity:
```

Comment:
I don't know about this. Why is it removed?

Reply:
This can have pretty dramatic effects unless I misunderstand what happens when you apply this as given by: https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#failure-domainbetakubernetesiozone